### PR TITLE
CD for formsflow web

### PR DESCRIPTION
**Pull request by sreehari-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Review by abhilash-aot — APPROVED** [review]

File: .github/workflows/forms-flow-web-cd.yml [hunk 1]
```diff
@@ -1,92 +1,34 @@
-name: Push Forms flow Web to registry
-
+name: forms-flow-web-CD
```
Comment (sumesh-aot) [inline .github/workflows/forms-flow-web-cd.yml:1]: Wouldn't this replace the CD action where we build and push the image ?

**Review by sumesh-aot — COMMENTED** [review]
I think this CD should be a separate one, as we still need to build the base image after building and pushing the compiled files to S3.

File: .github/workflows/forms-flow-web-cd.yml (hunk 1 above)
Comment (sreehari-aot) [reply .github/workflows/forms-flow-web-cd.yml:1]: yes, since we do not need the image anymore

Comment (sreehari-aot) [reply .github/workflows/forms-flow-web-cd.yml:1]: we only need the image for the root config.

**Review by sumesh-aot — APPROVED** [review]

**Comment by sumesh-aot** [comment]
Can you update version to 5.2.0-alpha, else it's will build 5.1.0 image and would fail for people who are using 5.1.0 now.
https://github.com/AOT-Technologies/forms-flow-ai/blob/5.2.0-alpha/VERSION

**Comment by sreehari-aot** [comment]
> Can you update version to 5.2.0-alpha, else it's will build 5.1.0 image and would fail for people who are using 5.1.0 now. https://github.com/AOT-Technologies/forms-flow-ai/blob/5.2.0-alpha/VERSION

sure will do

**Pull request merged**